src/adapters/refract/transformResource.coffee

```diff
@@ -40,9 +40,9 @@ module.exports = (resourceElement, location, options) ->
     resourceParameters = getUriParameters(_.get(resourceElement, 'attributes.hrefVariables'), options)
     actionParameters = getUriParameters(_.get(transitionElement, 'attributes.hrefVariables'), options)
 
-    attributes = _.dataStructures(transitionElement)
-    attributes = if _.isEmpty(attributes) then _.dataStructures(resourceElement) else attributes[0]
+    attributes = _.dataStructures(resourceElement)
```

Comment:
Basically, transitionElement will never have dataStructure element in it's content. So, I am just removing dead code which will never run.